packages/account/src/providers/provider.ts

```diff
@@ -1330,6 +1362,25 @@ Supported fuel-core version: ${supportedVersion}.`
     return coins;
   }
 
+  /**
+   * Returns an array of blobIds if they exist on chain, for a given array of blobIds.
+   *
+   * @param blobIds - blobIds to check.
+   * @returns - A promise that resolves to an array of blobIds that exist on chain.
+   */
+  async getBlobs(blobIds: string[]): Promise<(string | null)[]> {
```

Comment:
```suggestion
  /**
   * Returns an array of blobIds that exist on chain, for a given array of blobIds.
   *
   * @param blobIds - blobIds to check.
   * @returns - A promise that resolves to an array of blobIds that exist on chain.
   */
  async getBlobs(blobIds: string[]): Promise<string[]> {
```

The `return blobs.filter((v) => v);` below filters out all `null`s so we can return a `string[]` type.